### PR TITLE
I18n resource bundle

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [na] The description lists all applicable issues this PR seeks to resolve
- [na] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

This PR introduces a resource bundle for supporting multiple languages for trip notifications, starting with file `Message.java` which is adapted from the OTP source code.

### CODE REVIEW

File: src/main/java/org/opentripplanner/middleware/i18n/Message.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+import java.util.Locale;
+import java.util.MissingResourceException;
+import java.util.ResourceBundle;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The purpose of Messages is to read supply Message.properties to underlying calling code... The
```
Comment (br648) [inline src/main/java/org/opentripplanner/middleware/i18n/Message.java]: ... read AND supply...?

**Review by br648 — APPROVED** [review]
One minor comment, apart from that, tidy update!

File: src/main/java/org/opentripplanner/middleware/i18n/Message.java (hunk 1 above)
Comment (binh-dam-ibigroup) [reply src/main/java/org/opentripplanner/middleware/i18n/Message.java]: Alright, I updated the JavaDoc in https://github.com/ibi-group/otp-middleware/pull/217/commits/be6d19216957e6a70561ed146da9e147820cfd54.

File: src/main/java/org/opentripplanner/middleware/models/TripMonitorAlertNotification.java [hunk 2]
```diff
@@ -69,52 +83,60 @@ public static TripMonitorAlertNotification createAlertNotification(
...
+        if (hasNewAlerts && hasResolvedAlerts) {
+            return String.format(Message.TRIP_ALERT_NEW_AND_RESOLVED.get(locale), newAlertsText, resolvedAlertsText);
+        } else if (hasNewAlerts) {
+            return newAlertsText;
+        } else if (hasResolvedAlerts) {
+            return resolvedAlertsText;
         }
-        result.append(String.format(" %s", alertsWord));
-        return result.toString();
+        return "";
```
Comment (miles-grant-ibigroup) [inline src/main/java/org/opentripplanner/middleware/models/TripMonitorAlertNotification.java:140]: Do we not want an English default?

File: src/main/java/org/opentripplanner/middleware/models/TripMonitorNotification.java [hunk 3]
```diff
@@ -13,6 +14,7 @@
  */
 public class TripMonitorNotification extends Model {
     private static final Logger LOG = LoggerFactory.getLogger(TripMonitorNotification.class);
+    public static final String STOPWATCH_ICON = "⏱";
```
Comment (miles-grant-ibigroup) [inline src/main/java/org/opentripplanner/middleware/models/TripMonitorNotification.java:17]: Assuming the strings aren't unicode?

**Review by miles-grant-ibigroup — APPROVED** [review]
Yeessh this is finicky! Trusting you that this is the best option. Those hardcoded URLs make me nervous, but I think they're stable enough we don't have to make them configurable, right?

File: src/main/java/org/opentripplanner/middleware/models/TripMonitorAlertNotification.java (hunk 2 above)
Comment (binh-dam-ibigroup) [reply src/main/java/org/opentripplanner/middleware/models/TripMonitorAlertNotification.java:140]: For when there are no alerts?

File: src/main/java/org/opentripplanner/middleware/models/TripMonitorNotification.java (hunk 3 above)
Comment (binh-dam-ibigroup) [reply src/main/java/org/opentripplanner/middleware/models/TripMonitorNotification.java:17]: Which portion would be not-Unicode?

File: src/main/java/org/opentripplanner/middleware/models/TripMonitorAlertNotification.java (hunk 2 above)
Comment (miles-grant-ibigroup) [reply src/main/java/org/opentripplanner/middleware/models/TripMonitorAlertNotification.java:140]: Ah confused myself sorry!

File: src/main/java/org/opentripplanner/middleware/models/TripMonitorNotification.java (hunk 3 above)
Comment (miles-grant-ibigroup) [reply src/main/java/org/opentripplanner/middleware/models/TripMonitorNotification.java:17]: I guess I'm curious why we don't put the stop watch icon into the localized string?

Comment (binh-dam-ibigroup) [reply src/main/java/org/opentripplanner/middleware/models/TripMonitorNotification.java:17]: I guess it was a case of not repeating the icon, as in the trip alert notifications.

Comment (miles-grant-ibigroup) [reply src/main/java/org/opentripplanner/middleware/models/TripMonitorNotification.java:17]: Not a blocker. Happy to merge

**Pull request merged**